### ISSUE TITLE
Prepare for localization

### ISSUE THREAD

**Issue by jkotlinski** [jkotlinski/lsdpatch#43]
It would be great if this program could be translated to e.g. Japanese and Spanish.

**Comment by Eiyeron** [comment]
So, I see two things falling under the same scope : making the UI fully flexible to fit any size of text in any actionable, and making/using a localization data structure or library.

Sounds big enough to become a milestone, yes.

**Comment by jkotlinski** [comment]
Java provides ResourceBundle for localization, I think it makes sense to
use it.
We could make a few ones with Google Translate and see how it looks.

**Comment by Eiyeron** [comment]
I'd rather request specific people for translation help than leaving it to GTranslate. I fear it'd fail hard on GUI. From experience, it doesn't catch the proper term based on context.

I'd give a try for a french translation, but I think we would require the help of some people for the other languages.

**Comment by jkotlinski** [comment]
Yes, the Google Translate would be more to verify that the GUI accommodates different string lengths :-)
